### PR TITLE
Set static & dhcp sent ntp server ip on ethernet obj

### PR DESCRIPTION
**Pull request by asmithakarun**
Currently there is no way to differentiate between the Static
and DHCP assigned NTP servers at dbus

This commit adds StaticNTPServers peroperty to hold the static
configuration of NTP servers. This is a read-write property.
The existing NTPServer propery will hold both static and DHCP
assigned NTP servers. This is a read-only property.

The static and dhcp sent ntp server details are present in the
dbus object hosted by systemd-timesyncd service.

In this change, networkd fetches the ntp server details from the
timesyncd dbus object and set the "NTPServers" property with dhcp sent
ip and "StaticNTPServers" will be set from the configuration file.

Signed-off-by: Asmitha Karunanithi <asmitk01@in.ibm.com>
Change-Id: If8b1b3ed2afb531f179e00da173f6f86a1bf0c18

### CODE REVIEW

**Review by sunharis — CHANGES_REQUESTED** [review]
Posted some comments upstream. Please pull them here.

**Review by sunharis — APPROVED** [review]
Looks good. Please edit the commit message here as well - to make it a bit shorter

**Comment by raviteja-b** [comment]
Each Ethernet interface has NTPServers property and one ethernet interface can have static IP config and another interface can have DHCP enabled.
so DHCP provided NTP servers applies to specific interface 
How do we differentiate and populate NTP servers specific to each interfaces?

busctl introspect xyz.openbmc_project.Network /xyz/openbmc_project/network/eth0                                 
.NTPServers                                       property  as        0

**Comment by gtmills** [comment]
Upstream has a -1 from a maintainer, can we address Ratan's feedback, keep going upstream a bit to ensure we are headed in the correct direction before we merge this ?

**Comment by asmithakarun** [comment]
Created a new PR with these changes, since this fork doesnt contain the latest 1020 changes, and gerrit commit had a few more changes on top of it:
https://github.com/ibm-openbmc/phosphor-networkd/pull/34

**Pull request closed without merging**
